buildchain/buildchain/salt_tree.py

```diff
@@ -194,6 +194,8 @@ def task__deploy_salt_tree() -> Iterator[types.TaskDict]:
     Path('salt/metalk8s/macro.sls'),
     Path('salt/metalk8s/map.jinja'),
 
+    Path('salt/metalk8s/monitoring/prometheus-operator/deployed.sls'),
```

Comment:
I think we want monitoring to be a part of `addons`, but not sure actually. @NicolasT @TeddyAndrieux ?

Reply:
Yeah, I was not sure where to put this stuff.